### DISCUSSION TITLE
【测试】【高危】volumeshader_bm

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#1628] [其他]
测试你的H.D.D.性能
![image](https://github.com/user-attachments/assets/3979504a-5406-4b39-b3d0-654c4e81b113)

[点我开始](https://cznull.github.io/vsbm)

### 难度增加/降低方法：

点左上角CONFIG

![image](https://github.com/user-attachments/assets/09110eb7-b7f3-4469-a9e9-d2736c83fa0d)

修改for循环的次数（默认为5）

**Comment by RETRN1** [comment]
谢谢，HDD炸了（物理）😓

**Comment by BaCktOZwero** [comment]
群众里面有坏人😡（

**Comment by lllliufeng** [comment]
谢谢，GPU满了

**Comment by bugmaibear** [comment]
谢谢，默认设置已经把我的绳网搞闪退了😅

**Comment by laoyee234** [comment]
这使我的H.D.D.感觉身体不适

**Comment by adsadk12** [comment]
没压力

**Comment by aizai1173** [comment]
对不起做不到

**Comment by cpu644** [comment]
![image](https://github.com/user-attachments/assets/4441157e-2721-44fa-aaaf-345e80f54c04)
gt610开一个就吃满了

**Reply by Rain-YYC** [reply]
10gb显存的gt610.....果真疯牛病！

**Comment by RXLX-CHINA** [comment]
15可以稳定跑满，再多的话Fairy就被数据灌溢出来了

**Comment by loogerpin** [comment]
有点卡

**Comment by shilijia-siweikongjian** [comment]
不懂就问，这是测试机械硬盘（真•hdd）的，还是测试整个电脑的

**Reply by CloudSwordSage** [reply]
显卡或测试渲染的，基本都是显卡在跑，其他的没见动

**Comment by CloudSwordSage** [comment]
40 的时候开始感觉到了点击延迟，但在肉眼看来不仔细看还是看不出来，100 的时候有大概 0.8s 左右的点击延迟，不过我这只要调用gpu，基本都能满（1 的时候也是满的，习惯了，跑模型的时候也这样）
---
![image](https://github.com/user-attachments/assets/284eed18-d616-4510-aba6-35e8523e08a3)

**Comment by hfwang132** [comment]
怎么调用 gpu 跑啊，我在 edge 设置里已经打开了硬件加速，但是 gpu 还是 0%

**Comment by 1451374815** [comment]
4060表示问题不大

**Comment by scrooge-0** [comment]
4060刚好吃满，但还是有点伤机子

**Comment by arez-17** [comment]
卡死了!!
